src/pages/landing/[landingPage]/index.tsx

```diff
@@ -26,7 +26,8 @@ const LandingPage = ({
   const { pageTitle, pageDescription, documents, collections, articles } =
     landingPage
 
-  const pageContent = (): JSX.Element => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const pageContent = (): any => {
```

Comment:
The content prop for the `InPageNavigation` component expects a type of `Element & string` for some reason that I don't understand. I tried to remedy this a few other ways, but nothing else worked so I added this `any` and explicitly ignored the type.